src/doc-templates/class.md.jinja2

```diff
@@ -42,7 +42,7 @@ _{{ element_description_line }}_
 {{ gen.link(element.mixins[0]) }} [Checklist] + {{ gen.link(element.is_a) }} [Extension]
 {% endif %}
 
-## Terms
+## Terms {{ title }} {% if element.deprecated %} <span style="color: red;"><strong> (DEPRECATED) </strong> {% endif %}
```

Comment:
@sujaypatil96 is this right???
Or does it need to go on line 21?

Reply:
Yup, this needs to go on line 21.

Here: https://github.com/GenomicsStandardsConsortium/mixs/blob/9da8adf6eae045fd8db5fe138c4f8eb304041cb8/src/doc-templates/class.md.jinja2#L21

Reply:
Line 21 only? So I should delete it from line 45?

Reply:
Fixed? @sujaypatil96

Reply:
Yup, this change looks good!